config/database/config.go

```diff
@@ -0,0 +1,67 @@
+package database
+
+import (
+	"time"
+
+	"github.com/flyteorg/flytestdlib/config"
+	"gorm.io/gorm/logger"
+)
+
+const (
+	database = "database"
+	postgres = "postgres"
+)
+
+var Config = config.MustRegisterSection(database, &DbConfig{
```

Comment:
We shouldn't use the deprecated fields in here... let's use the new fields only...

Reply:
Updated it